cognition_objects/pipeline_version.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Optional, Dict, Any
+from ..business_objects import general
+from ..session import session
+from ..models import CognitionPipelineVersion
+from .. import enums
+
+MAX_AUTO_SAVE_VERSIONS = 10
+
+
+def get_current_version(project_id: str) -> Optional[CognitionPipelineVersion]:
+    return (
+        session.query(CognitionPipelineVersion)
+        .filter(
+            CognitionPipelineVersion.project_id == project_id,
+        )
+        .order_by(CognitionPipelineVersion.created_at.desc())
```

Comment:
Is the assumption current -> created_at desc always true? If we set our current version to some older version e.g.

Reply:
yes it's true (also a reason why the activate creates a new version :)